src/components/DateRangePicker.jsx

```diff
@@ -405,6 +408,8 @@ export default class DateRangePicker extends React.Component {
             <CloseButton />
           </button>
         }
+
+        {bottomPanel}
```

Comment:
Maybe put this inside a namespaced `<div className="DateRangePicker__bottom-panel">...</div>` or something?